packages/editor/src/store/actions.js

```diff
@@ -195,23 +195,38 @@ export const savePost =
 				),
 			content,
 		};
-		dispatch( { type: 'REQUEST_POST_UPDATE_START', options } );
-		await registry
-			.dispatch( coreStore )
-			.saveEntityRecord(
-				'postType',
-				previousRecord.type,
+		let error;
+		try {
+			edits = await applyFilters(
+				'editor.__unstablePreSavePost',
 				edits,
```

Comment:
Let's pass `Promise.resolve( edits )` as the filter argument, not just the plain `edits`. Then the filter function can rely on the first argument being a promise, and call `edits.then` without fear.

Reply:
fixed in ff7d6b430a139ae0ffcb77d64796b8c6919ffe2c